content/ordering.ts

```diff
@@ -176,8 +176,8 @@ const MODULE_ORDERING: { [key in SectionID]: Chapter[] } = {
       items: ['bin-jump', 'merging', 'hld', 'centroid'],
     },
     {
-      name: 'Convex Hull',
-      description: 'Most Platinum geometry problems.',
+      name: 'Geometry',
+      description: 'More advanced concepts in computational geometry.',
```

Comment:
> More advanced

more advanced than what? Geometry hasn't appeared earlier in the guide :)

Reply:
There's rectangle geometry in bronze

Reply:
fair enough